### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by SRazvan**

### CODE REVIEW

File: Students/Stoica S. Razvan Cosmin/Assignment1/assign3/src/assign3/assign3.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+			inv = inv * 10 + aux % 10;
+			aux = aux / 10;
+		}
+		if (n == inv)
+			return 1;
+		else
+			return 0;
+	}
+
+	public static void Palindrom() {
```
Comment (georgebejan) [inline Students/Stoica S. Razvan Cosmin/Assignment1/assign3/src/assign3/assign3.java]: According to [this](http://www.oracle.com/technetwork/java/codeconventions-135099.html) the first letter of a method name should be lowercase.

File: Students/Stoica S. Razvan Cosmin/Assignment1/assign3/src/assign3/assign3.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package assign3;
+
+public class assign3
```
Comment (georgebejan) [inline Students/Stoica S. Razvan Cosmin/Assignment1/assign3/src/assign3/assign3.java]: According to [this](http://www.oracle.com/technetwork/java/codeconventions-135099.html) class name should have the first letter uppercase

File: Students/Stoica S. Razvan Cosmin/Assignment1/asign/src/asign/asign2.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package asign;
+
+import java.util.Scanner;
+
+public class asign2 {
```
Comment (georgebejan) [inline Students/Stoica S. Razvan Cosmin/Assignment1/asign/src/asign/asign2.java]: According to [this](http://www.oracle.com/technetwork/java/codeconventions-135099.html) class name should have the first letter uppercase

**Comment by Alex-Cosma** [comment]
Good work!

**Pull request merged**